doc/manual/manual.tex

```diff
@@ -5023,6 +5023,34 @@ \subsubsection{Prescribed internal velocity constraints}
     \label{fig:prescribed-velocity}
 \end{figure}
 
+\subsubsection{Artificial viscosity smoothing}
+\label{sec:artificial-viscosity-smoothing}
+\textit{This section was contributed by Ryan Grove}\\
+
+Standard finite element discretizations of advection-diffusion equations introduce unphysical oscillations around steep gradients. Therefore, stabilization must be added to the discrete formulation to obtain correct solutions. In ASPECT, we use the Entropy Viscosity scheme developed by Guermond et al. in the paper \cite{guer11}. In this scheme, an artificial viscosity is calculated on every cell in and used to try to combat these oscillations that cause unwanted overshoot and undershoot.  More information about how ASPECT does this is located at \url{https://dealii.org/developer/doxygen/deal.II/step_31.html}.  \\
+
+Instead of just looking at an individual cell's artificial viscosity, improvements in the minimizing of the oscillations can be made by smoothing.  The motivation behind smoothing came from the idea of not only using a particular cell's artificial viscosity, but the artificial viscosity of neighboring cells as well.  The smoothing process refers to the comparing of the artificial viscosity of each cell with the artificial viscosity of each of the particular cell's neighboring cells and the maximum of this comparison is then used for future calculations.\\
+
+This feature can be turned on by setting the $Use \text{  } artificial \text{  }  viscosity$ flag inside the $Stabilization \text{  } parameters$ subsection inside the $Discretization$ subsection in your parameter file.
```

Comment:
Instead of putting the text in `$...$`, let's provide a complete reference. This should look something like `hyperref[....]{Use artificial viscosity}` (is there a "smoothing" missing?) and you can find the exact text in the `.out` file that corresponds to your input file snippet.

Similarly, instead of `$Discretization$`, use `\hyperref[parameters:Discretization]{Discretization}`.